examples/runtime/starlark/inline/README.md

```diff
@@ -0,0 +1,43 @@
+# Starlark Inline Example
+
+In this example, we are going to demonstrate how to execute a starlark script
+in starlark function against the input KRM resources to update or validate them.
+
+The starlark script is provided inline in the function configuration like below:
+
+```
+apiVersion: fn.kpt.dev/v1beta1
```

Comment:
As in other PR, this should be followed by 1 or 2 sentences explain what this does.